deepchecks/vision/checks/performance/performance_report.py

```diff
@@ -214,9 +194,6 @@ def add_condition_class_performance_imbalance_ratio_not_greater_than(
             DeepchecksValueError:
                 if unknown score function name were passed;
         """
-        if score is None:
-            score = next(iter(MULTICLASS_SCORERS_NON_AVERAGE))
```

Comment:
Need some logic to get the default scorer in this condition....

Reply:
Condition doesn't work yet, and currently this is an undefined variable that crashes pylint

Reply:
so at least add a todo or task

Reply:
Done